src/main/java/ca/on/oicr/gsi/dimsum/util/reporting/reports/MohTglTrackingReport.java

```diff
@@ -0,0 +1,194 @@
+package ca.on.oicr.gsi.dimsum.util.reporting.reports;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import com.fasterxml.jackson.databind.JsonNode;
+import ca.on.oicr.gsi.cardea.data.Assay;
+import ca.on.oicr.gsi.cardea.data.Case;
+import ca.on.oicr.gsi.cardea.data.Metric;
+import ca.on.oicr.gsi.cardea.data.MetricCategory;
+import ca.on.oicr.gsi.cardea.data.MetricSubcategory;
+import ca.on.oicr.gsi.cardea.data.Sample;
+import ca.on.oicr.gsi.cardea.data.Test;
+import ca.on.oicr.gsi.dimsum.service.CaseService;
+import ca.on.oicr.gsi.dimsum.service.filtering.CaseFilter;
+import ca.on.oicr.gsi.dimsum.service.filtering.CaseFilterKey;
+import ca.on.oicr.gsi.dimsum.service.filtering.CompletedGate;
+import ca.on.oicr.gsi.dimsum.util.reporting.Column;
+import ca.on.oicr.gsi.dimsum.util.reporting.Report;
+import ca.on.oicr.gsi.dimsum.util.reporting.ReportSection;
+import ca.on.oicr.gsi.dimsum.util.reporting.ReportSection.StaticTableReportSection;
+
+public class MohTglTrackingReport extends Report {
+
+    private static final String METRIC_COVERAGE = "Mean Coverage Deduplicated";
+    private static final String METRIC_CLUSTERS = "Pipeline Filtered Clusters";
+
+     record RowData(Case kase, Test test, Assay assay, Sample sample) { }
+
+    private static final ReportSection<RowData> trackerSection =
+            new StaticTableReportSection<RowData>("Tracker",
+                    Arrays.asList(
+                            Column.forString("Case ID", x -> x.kase().getId()),
+                            Column.forString("Assay", x -> x.kase().getAssayName()),
+                            Column.forString("External Name", x -> x.kase().getDonor().getExternalName()),
+                            Column.forString("Test", x -> x.test().getName()),
+                            Column.forString("Tissue Type", x -> x.test().getTissueType()),
+                            Column.forString("Group ID", x -> x.test().getGroupId()),
+                            Column.forString("Extraction Pass?",
+                                    x -> CompletedGate.EXTRACTION.qualifyTest(x.test()) ? "Yes" : null),
+                            Column.forString("Stock ID", x -> x.test().getExtractions().stream()
+                                    .map(Sample::getId).collect(Collectors.joining(", "))),
+                            Column.forString("Stock Name (Sample Alias)", x -> x.test().getExtractions().stream()
+                                    .map(Sample::getName).collect(Collectors.joining(", "))),
+                            Column.forString("Library (Tissue Alias)", x -> x.sample().getName()),
```

Comment:
* We don't use the term "alias" in Dimsum
* Library and tissue are two different things, so I'm not sure what you actually want here
* `x.sample()` is a full-depth sequencing sample, which is a run-library, so the name you're getting is the library aliquot name